### DISCUSSION TITLE
Best tool to model sprinkler attenuation

### DISCUSSION THREAD

**Discussion by dmmcmahon-lanl** [firemodels/cfast#2067] [General]
What is the best modeling tool (CFAST vs. FDS) to use for attenuation of heat release rate after sprinkler activation? I compared CFAST with FDS for the same problem and got very different results. 

Thanks

**Comment by drjfloyd** [comment]
How do you define "best"?  Which model is better will depend on the specific usage of the model, the uncertainty you are willing to tolerate, the types of outputs you are interested in, and the resources you have available. 

Both CFAST and FDS have a suppression model where the burning rate decreases exponentially based on the net amount of water delivered to the fire.  The implemenations are not idenitical.  CFAST does not evaporate droplets or have any interactions between the spray and the layer.
-CFAST uses the AVERAGE delivered density over the compartment floor and the empirical constant used in the fire decay is fixed.
-FDS uses the ACTUAL delivered density to adjust each wall cell independently for those wall cells that have a SURF wtih an HRRPUA and an E_COEFFICIENT that you must define. You could define in FDS the same constant as used in CFAST, however, it is very unlikely that you have defined a spray pattern that delivers exactly the same average spray density to all burning wall cells at all times.  

You should also consider that the Evans sprinkler model is based on a specific set of experiments and that the resulting decay of the fire in the Evans model may or may not be applicable to your specific scenario. A better question than is CFAST or FDS best, may be is it even justified to use the Evans model vs. just imposing a speciifc heat release profile over time at sprinkler activation.
